bk-api/bk_api.py

```diff
@@ -852,8 +860,34 @@ def deploy_wes(node_id, this_debug, force=False):
 
     return {"success":True}
 
+def add_vsn_event(node_id,field_value, lock_tables=True, lock_requested_by=""):
+
+    payload = {"node_id": node_id, "source": "beekeeper-add-vsn", "operation":"insert",
```

Comment:
can you please make sure that the node_id is all upper-case?

I think we want to normalize all the VSN values to be the same case so that `w01a` becomes `W01A`.

Reply:
Checks for node-id is all upper-case is redundant since there are a lot of try-catches upstream and downstream. Also it matches the same logic as the other functions since in none of them there are checks for upper-case. I can create a new ticket to add it in the future.

yes we can normalize vsn to be all upper-case. thats easy.

Reply:
yeah i dont want checks.  sorry, that was unclear. i just wanted you to normalize it to upper() and be done with it.